org.freedesktop.Sdk.Extension.llvm18.json

```diff
@@ -0,0 +1,177 @@
+{
+  "id": "org.freedesktop.Sdk.Extension.llvm18",
+  "branch": "23.08",
+  "runtime": "org.freedesktop.Sdk",
+  "build-extension": true,
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "23.08",
+  "separate-locales": false,
+  "appstream-compose": false,
```

Comment:
You can drop this and manual compose, it should work out of the box now.

Reply:
Do you mean entire:
```
    {
      "name": "appdata",
      "buildsystem": "simple",
      "build-commands": [
        "install -Dm0644 -t ${FLATPAK_DEST}/share/metainfo/ ${FLATPAK_ID}.metainfo.xml",
        "appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}"
      ],
```

can be dropped?

Reply:
No keep the install part drop appstream-compose command and appstream-compose: false

Reply:
done